packages/react-components/review/beta/react-components.api.md

```diff
@@ -653,6 +653,7 @@ export interface ComponentStrings {
     raiseHandButton: RaiseHandButtonStrings;
     // @beta
     reactionButton: ReactionButtonStrings;
+    rteSendBox: RTESendBoxStrings;
```

Comment:
Since these strings and icons will be part of the public API, should we have them as richTextEditor instead of rte?

Reply:
I'm a little bit worried that the names will be too long. 
Also, if we are going to leave the send box with the rich text editor component name as `RTESendBox`, the strings name will be consistent

Reply:
Updated